supplier_management_site/tests/test.py

```diff
@@ -22,3 +32,18 @@ def test_testing_page_contains_correct_translation(self, lang, text):
         request = self.factory.get("/{}/".format(lang))
         response = home(request)
         self.assertContains(response, text)
+
+
+class EmailUtilsTest(TestCase):
+    def tearDown(self):
+        mail.outbox = []
```

Comment:
Why this tearDown? You have one single test

Reply:
I left it there by mistake 😞